src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/service/RecallService.kt

```diff
@@ -294,6 +295,24 @@ class RecallService(
     )
   }
 
+  private fun deleteDpsRecallSentence(recallSentence: RecallSentenceEntity) {
+    recallSentence.preRecallSentenceStatus?.let { preRecallSentenceStatus ->
+      recallSentence.sentence.statusId = preRecallSentenceStatus
+      recallSentence.sentence.updatedAt = ZonedDateTime.now()
+      recallSentence.sentence.updatedBy = "DELETE RECALL" // TODO  pass in from caller to api - needs setting on the RecallEntity too
```

Comment:
currently the username and prison are notpassed through, will follow up with another ticket